compiler/noirc_errors/src/debug_info.rs

```diff
@@ -85,4 +99,11 @@ impl DebugInfo {
 
         counted_opcodes
     }
+
+    pub fn get_file_ids(&self) -> Vec<FileId> {
+        self.locations
+            .values()
+            .filter_map(|call_stack| call_stack.last().map(|location| location.file))
```

Comment:
I'm pretty sure we can't have empty callstacks so this filtering may be unnecessary.